core/pkg/evaluator/fractional.go

```diff
@@ -96,37 +111,32 @@ func parseFractionalEvaluationDistributions(values []any) ([]fractionalEvaluatio
 			return nil, errors.New("first element of distribution element isn't string")
 		}
 
-		percentage, ok := distributionArray[1].(float64)
+		weight, ok := distributionArray[1].(float64)
```

Comment:
@toddbaert you proposed to make the second element optional w/ `1` as default, is it still something that we want to do? 🤔

Reply:
I will adapt the PR to also support that use case - since this does not break anything it makes sense to also have this :)

Reply:
Yep!